webextension/aboutConfigPrefs.js

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+/* global ExtensionAPI, Services */
+
+ChromeUtils.import("resource://gre/modules/Services.jsm");
+ChromeUtils.import("resource://gre/modules/ExtensionCommon.jsm");
+ChromeUtils.defineModuleGetter(this, "AddonManager", "resource://gre/modules/AddonManager.jsm");
+
+const EventManager = ExtensionCommon.EventManager;
+
+this.aboutConfigPrefs = class extends ExtensionAPI {
+  getAPI(context) {
+    const extensionIDBase = context.extension.id.split("@")[0];
+    const prefBranchPrefix = `extensions.${extensionIDBase}.`;
+    const prefBranch = Services.prefs.getBranch(prefBranchPrefix);
+    const prefChangeEventName = "experiments.aboutConfigPrefs.onPrefChange";
+
+    function get(type, name) {
+      try {
+        return prefBranch[`get${type}Pref`](name);
+      } catch (_) {
+        return undefined;
+      }
+    }
+
+    const prefsToClearOnUninstall = new Set();
+
+    return {
+      experiments: {
+        aboutConfigPrefs: {
+          onPrefChange: new EventManager(context, prefChangeEventName, (fire, name) => {
```

Comment:
EventManager changed in bug 1450388 (which landed in 61) to take an object with a bunch of named properties instead of positional arguments.  We have a compatibility shim in place for now, but at some point that shim will be removed so if/when you're no longer targeting users running 60 or older, this should be updated to use the new style.

Reply:
Alright. It looks like we'll just have to rely on the shim for the duration of this experiment, since I can't simply try both variants (I tried, and in v60 it doesn't seem to throw a catchable error when I pass in named props, though one is logged to console: `this.register is not a function`). I'll file an issue to migrate the code over for when the shim is being removed.